rtcstats.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 (function() {
+  // transforms a maplike to an object. Mostly for getStats +
+  // JSON.parse(JSON.stringify())
+  function m2o(m) {
```

Comment:
That's not really a reason not to do it though, jQuery has readable source code, and minifies it for inclusion in script tags :P I say it because splitscreen.js is very bad mostly because at the beginning most of the source code for it was written like this. I don't care much about the content, it's a simple function to convert, but the name here needs to be more descriptive imo. Otherwise I'm just going to assume you're referencing http://www.m2o.it/m2o/home ;)